website/docs/reference/unleash-context.md

```diff
@@ -40,7 +40,7 @@ As an example: You've created a custom field called `groupId`. You know group ID
 
 
 
-## Custom context fields
+## Custom context fields {#context-fields}
```

Comment:
Out of curiosity: why did you change the id here? Have you checked whether that breaks any other links to this section from within the docs?

Reply:
I though it's not created automatically without this. Reverting this change as I see that's not the case and I can use #custom-context-field